openapi.yaml

```diff
@@ -9423,11 +9418,12 @@ paths:
           content:
             application/json:
               schema:
-                type: object
-                properties:
-                  secret_key:
-                    type: string
-                    example: OiA6F5r0niLs3QA2stbyq7mY5VCV7KqOzcmitmHw
+                allOf:
```

Comment:
it seems like we do need to have the full schema for an object storage key here. In either case, the response isn't showing the contents of the schema, but that seems like a front-end bug?

Reply:
Yeah - I think Alban is going t work on this. I'm unable to get this response to show. There are other cases of this as well.